packages/renderer/src/lib/appearance/Appearance.spec.ts

```diff
@@ -30,16 +30,21 @@ import Appearance from './Appearance.svelte';
 
 const getConfigurationValueMock = vi.fn();
 const addEventListenerMock = vi.fn();
+const matchMediaMock = vi.fn();
+
+beforeAll(() => {
+  Object.defineProperty(window, 'getConfigurationValue', { value: getConfigurationValueMock });
+  Object.defineProperty(window, 'matchMedia', { value: matchMediaMock });
```

Comment:
non blocking: this method is now globally exposed so I would say it can be removed